tests/bfd/test_bfd.py

```diff
@@ -330,6 +338,21 @@ def update_bfd_state(ptfhost, neighbor_addr, local_addr, state):
     ptfhost.shell("bfdd-control session local {} remote {} {}".format(neighbor_addr, local_addr, state))
 
 
+def verify_bfd_queue_counters(duthost, dut_intf):
+    queue_output = duthost.shell("show queue counters {}".format(dut_intf))
+    logger.debug("Queue output: {}".format(queue_output['stdout']))
+
+    for queue_val in range(0, 7):
+        queue_pkt_count, _ = get_egress_queue_count(duthost, dut_intf, int(queue_val))
+        logger.debug("Interface {}, Queue {}, counter {}".format(dut_intf, queue_val, queue_pkt_count))
+        if queue_pkt_count != 0:
+            pytest.fail('Queue {} count is not zero, BFD packets might use this'.format(queue_val))
+
+    bfd_queue = 7
+    queue_pkt_count, _ = get_egress_queue_count(duthost, dut_intf, int(bfd_queue))
```

Comment:
Can we assert that queue_pkt_count is not zero as well and fail the test otherwise?

Reply:
Done